content/docs/tutorials/dess/aws/7-activation/index.md

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
Just a note for working with git:
This file appeared as an add & delete rather than a rename, make sure to add both the new and old version of the file to your staged changes when you commit. It makes it easier for me to see the actual changes.

Reply:
Hm I was able to do that for all the other commits just not this one for some reason